### ISSUE TITLE
Jakt compiler hangs when compiling function

### ISSUE THREAD

**Issue by mnlrsn** [SerenityOS/jakt#321]
This makes the compiler hang (possibly an infinite loop somewhere):

```
function make_array_f64 throws -> [f64] {
    let a : [f64] = [0f64]
    return a
}
```

**Comment by linusg** [comment]
This is missing `()` after the function name

**Comment by gdelazzari** [comment]
This is also fixed by #278

**Comment by mnlrsn** [comment]
> This is missing `()` after the function name

> This is also fixed by #278

Oops - and thanks for looking into this. So the compiler is not quite idiot proof yet...

**Comment by mnlrsn** [comment]
Indeed fixed, possibly by #278
